lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -3915,14 +3820,18 @@ class PullbackEmitter final : public SILInstructionVisitor<PullbackEmitter> {
   DenseMap<std::pair<SILBasicBlock *, SILValue>, SILArgument *>
       activeValuePullbackBBArgumentMap;
 
-  /// Mapping from original basic blocks to local adjoint values to be cleaned
-  /// up. This is populated when pullback generation is run on one basic block
-  /// and cleaned before processing another basic block.
-  DenseMap<SILBasicBlock *, SmallVector<AdjointValue, 8>>
-      blockLocalAdjointValues;
+  /// Mapping from original basic blocks to local temporary values to be cleaned
+  /// up. This is populated when adjoint emission is run on one basic block and
+  /// cleaned before processing another basic block.
+  DenseMap<SILBasicBlock *, SmallVector<SILValue, 64>>
```

Comment:
Curious why the bump from initial vector size from 8 to 64? Because very few basic blocks will have 8?

Reply:
Many functions being differentiated are huge and have a huge number of values. Starting off with a large size is better than resizing multiple times from a small size.